movieApp.js

```diff
@@ -1,204 +1,279 @@
+    stars.forEach((star, clickedStarIndex) => {
+      star.addEventListener("click", () => {
+        stars.forEach((star, starIndex) =>
+          starIndex <= clickedStarIndex
+            ? (star.className = starClassActive)
+            : (star.className = starClassInactive)
+        );
       });
-    };
-  });
+    });
 
-  movieContainer.appendChild(img);
-  movieContainer.appendChild(readMoreBtn);
-  movieContainer.appendChild(starRating);
+    movieContainer.appendChild(img);
+    movieContainer.appendChild(readMoreBtn);
+    movieContainer.appendChild(starRating);
+    movieDataContainer.appendChild(movieContainer);
 
-  // Create element to show details about the movie
-  const movieDetails = document.createElement("div");
-  movieDetails.className = "movie-details";
-  movieDetails.style.display = "none";
-  movieDetails.style.color = "white";
-  movieDetails.innerHTML = `
+    // Create element to show details about the movie
+    const storeMovieDetail = document.createElement("div");
+    storeMovieDetail.className = "storeMovieDetail";
+    const movieDetails = document.createElement("div");
+    movieDetails.className = "movie-details";
+    movieDetails.style.display = "none";
+    movieDetails.style.color = "white";
+    movieDetails.innerHTML = `
            <p><strong>Title:</strong> ${movie.title}</p>
            <p><strong>Description:</strong> ${movie.description}</p>
            <p><strong>Release Year:</strong> ${movie.movie_year}</p>
            <p><strong>Director:</strong> ${movie.director}</p>
            <p><strong>Actors:</strong> ${movie.actors.join(", ")}</p>
        `;
-  movieContainer.appendChild(movieDetails);
-
-  // Adding the comment Section
-
-  function createCommentSection() {
-    const commentSection = document.createElement("div");
-    commentSection.innerHTML = "Comments: ";
-    commentSection.style.fontWeight = "bold";
-    commentSection.className = "comment-section";
-    const commentInput = document.createElement("input");
-    commentInput.type = "text";
-    commentInput.value = "";
-    commentInput.placeholder = "Add a Comment";
-    commentInput.style.color = "white";
-    const commentButton = document.createElement("button");
-    commentButton.className = "comment-button";
-    commentButton.innerHTML = "Submit";
-    const comments = document.createElement("div");
-    comments.className = "comments";
-    const commentUl = document.createElement("ul");
-
-    commentButton.addEventListener("click", () => {
+    storeMovieDetail.appendChild(movieDetails);
+    movieDataContainer.appendChild(storeMovieDetail);
+
+    // Adding the comment Section
+
+    function createCommentSection() {
+      const commentSection = document.createElement("div");
+      commentSection.innerHTML = "Comments: ";
+      commentSection.style.fontWeight = "bold";
+      commentSection.className = "comment-section";
+      const commentInput = document.createElement("input");
```

Comment:
Try to add an empty line above each part of your function code to make it easier to read :)